functions/openhab.js

```diff
@@ -151,6 +168,59 @@ function getItemsState(request,response) {
 		})
 }
 
+/**
+ * Gets Temperature or Thermostat Data
+ */
+function getTempData(item) {
+	var thermData = {};
+	var thermItems = item.tags.toString().includes("Thermostat") ? getThermostatItems(item.members) : getThermostatItems([item]);
+	
+	//Are we dealing with Fahrenheit?
+	var isF = item.tags.indexOf('Fahrenheit') >= 0 ? true : false;
+	
+	//store long json variables in easier variables to work with below
+	var tstatMode = thermItems.hasOwnProperty('heatingCoolingMode') ? (thermItems.heatingCoolingMode.state.length == 1 ? utils.normalizeThermostatMode(thermItems.heatingCoolingMode.state) : thermItems.heatingCoolingMode.state) : 'heat'
```

Comment:
It's easier to simply check `thermItems.heatingCoolingMode ? ... : ...` and so on.

Reply:
Well...I initially had it that way, but since some devices/items provide an integer for them mode (e.g. 0=off, etc.) I was trying to come up with a way to combine so that tstatMode would always provide what google is looking for (auto, heat, cool, heatcool, off).  Maybe it should be two lines? 1) figure out whether it is a number, 2) if it's a number go to utils function to change to proper string